garrysmod/gamemodes/terrortown/gamemode/lang/german.lua

```diff
@@ -12,30 +12,30 @@ L.terrorists = "Terroristen"
 L.spectators = "Zuschauer"
 
 --- Round status messages
-L.round_minplayers = "Nicht genug Spieler um eine neue Runde zu beginnen..."
-L.round_voting     = "Es läuft eine Umfrage, verzögere neue Runde um {num} Sekunden..."
-L.round_begintime  = "Eine neue Runde beginnt in {num} Sekunden. Bereite dich vor."
+L.round_minplayers = "Zu wenig Spieler um eine neue Runde zu beginnen..."
+L.round_voting     = "Es läuft eine Umfrage, verzögere neue Runde um {num} Sekunde(n)..."
+L.round_begintime  = "Eine neue Runde beginnt in {num} Sekunde(n). Bereite dich vor."
 L.round_selected   = "Die Verräter wurden ausgewählt."
 L.round_started    = "Die Runde hat begonnen!"
 L.round_restart    = "Ein Admin erzwang den Neustart der Runde."
 
-L.round_traitors_one  = "Verräter, du bist allein."
+L.round_traitors_one  = "Verräter, du bist alleine."
 L.round_traitors_more = "Verräter, dies sind die Namen deiner Verbündeten: {names}"
 
 L.win_time         = "Die Zeit ist abgelaufen. Die Verräter haben verloren."
 L.win_traitor      = "Die Verräter haben gewonnen!"
 L.win_innocent     = "Die Verräter wurden besiegt!"
-L.win_showreport   = "Schauen wir uns den Runden Report für {num} Sekunden an."
+L.win_showreport   = "Schauen wir uns den Runden Report für {num} Sekunde(n) an."
 
 L.limit_round      = "Rundenlimit erreicht. {mapname} wird bald geladen."
 L.limit_time       = "Zeitlimit erreicht. {mapname} wird bald geladen."
-L.limit_left       = "{num} Runde(n) oder {time} Minuten verbleibend bis die Map zu {mapname} gewechselt wird."
+L.limit_left       = "{num} Runde(n) oder {time} Minute(n) verbleibend bis die Map zu {mapname} gewechselt wird."
 
 --- Credit awards
 L.credit_det_all   = "Detektive, euch wurde(n) {num} Ausrüstungs-Credit(s) für eure Performance gegeben."
 L.credit_tr_all    = "Verräter, euch wurde(n) {num} Ausrüstungs-Credit(s) für eure Performance gegeben."
 
-L.credit_kill      = "Dir wurde(n) {num} Credit(s) für das Töten eines {role} gegeben."
+L.credit_kill      = "Dir wurde(n) {num} Credit(s), da du einen {role} getötet hast."
```

Comment:
Shouldn't that be: Dir wurde(n) {num} Credit(s) **gegeben**, da du einen {role} getötet hast. ?